assets/dev/js/editor/document/elements/commands/browser-import.js

```diff
@@ -0,0 +1,33 @@
+import CommandBase from 'elementor-api/modules/command-base';
```

Comment:
Why not merge 'browser-import.js' with 'import.js' and use 
`document/elements/import`
for all the situations?

Reply:
@matipojo what you decided it should be component `browser-import` or can @ItzhakAvm merge them?

Reply:
> @matipojo what you decided it should be component `browser-import` or can @ItzhakAvm merge them?

The `document/elements/import` is responsible for import via the server, but the `browser-import imports in the client only.